charts/drax/values.yaml

```
@@ -2126,58 +2153,131 @@ influxdb2:
       enabled: false
     # renovate:
     image:
-      repository: curlimages/curl
-      tag: "8.13.0"
+      repository: ubuntu
+      tag: "22.04"
```

We should at least take the latest LTS release. Probably it would be better to reuse the `acc-generic-img` (less images that need to be pulled and we just need a shell and curl which is already installed in there)?

```suggestion
    image:
      repository: accelleran/acc-generic-img
      tag: "0.9.0"
```